pkg/storaged/mdraid/mdraid.jsx

```diff
@@ -91,22 +91,27 @@ function mdraid_stop(mdraid) {
 }
 
 function mdraid_delete(mdraid, block, card) {
-    function delete_() {
-        if (mdraid.Delete)
-            return mdraid.Delete({ 'tear-down': { t: 'b', v: true } }).then(reload_systemd);
```

Comment:
So base udisks version required is now 2.0.0, and rhel-8-10 has 2.9.x! So this works